EuroPythonBot/registration/pretix_connector.py

```diff
@@ -61,6 +61,10 @@ async def _fetch_pretix_orders(self) -> None:
                 continue
 
             for position in order.positions:
+                # skip positions without name (e.g. childcare, T-shirt)
+                if position.attendee_name is None:
```

Comment:
```suggestion
                if not position.attendee_name:
```
Why is the comparison to None? Do we keep empty?
Are we sure there will always be an attendee_name in the order position?

Reply:
- `is None` explicitly checks the type: `position.attendee_name` is `str | None`, and `None` would cause an AttributeError later.
- Empty strings are not expected: Pretix order positions either have a non-empy `attendee_name`, or the ordered item cannot hold an `attendee_name` (`"attendee_name": null` in their API responses).
- Empty strings (i.e. `"attendee_name": ""` in an API response) would not cause an issue: The registration form [demands `min_length=1`](https://github.com/EuroPython/discord/blob/17123141eeea17bab807a1bd7dc32a2327ea556f/EuroPythonBot/registration/cog.py#L46), and if someone finds a way to bypass this check and knows the order ID of a ticket with an empty name, they deserve the Discord server access.

I guess this is a matter of perspective and personal preference. Changed in e5961c6304c764335a17873175e699c02c903fdc.